strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/npm/NpmArtifactControllerTestIT.java

```diff
@@ -123,39 +126,40 @@ public void testViewArtifactViaProxy(@Remote(url = REMOTE_URL)
                .body("name", equalTo("react"))
                .body("versions.size()", greaterThan(0));
 
-        ArtifactEntry artifactEntry = artifactEntryService.findOneArtifact(storageId,
-                                                                           repositoryId,
-                                                                           coordinates.toPath());
+        Artifact artifactEntry = artifactEntityRepository.findOneArtifact(storageId,
+                                                                          repositoryId,
+                                                                          coordinates.buildPath());
         assertThat(artifactEntry).isNotNull();
-        assertThat(artifactEntry).isInstanceOf(RemoteArtifactEntry.class);
-        assertThat(((RemoteArtifactEntry)artifactEntry).getIsCached()).isFalse();
+        assertThat(artifactEntry.getArtifactFileExists()).isFalse();
     }
 
     @ExtendWith(RepositoryManagementTestExecutionListener.class)
     @Test
+    @Disabled
     public void testSearchArtifactViaProxy(@Remote(url = REMOTE_URL)
                                            @NpmRepository(repositoryId = REPOSITORY_PROXY)
                                            Repository proxyRepository)
     {
         final String storageId = proxyRepository.getStorage().getId();
         final String repositoryId = proxyRepository.getId();
 
+        //TODO: we should be able to search by `reston` substring with `Mixed Index`
```

Comment:
So, what does this mean? Do we need a follow-up task for this? If so, could you please raise one? Thanks! :)

Reply:
#1980